### PR TITLE
subtitleedit: init at 3.6.11

### PR DESCRIPTION
**Pull request by paveloom**
###### Description of changes

1) Add myself to the maintainers list
2) Add a Mono wrapper for the portable version of [Subtitle Edit](https://nikse.dk/subtitleedit)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/subtitleedit/default.nix [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+stdenv.mkDerivation rec {
+  pname = "subtitleedit";
+  version = "3.6.11";
+
+  src = fetchzip {
+    url = concatStringsSep "/" [
+      "https://github.com/SubtitleEdit/subtitleedit/releases/download"
+      version
+      "SE${concatStrings (splitVersion version)}.zip"
+    ];
```
Comment (sikmir) [inline pkgs/applications/video/subtitleedit/default.nix:30]:
```suggestion
    url = "https://github.com/SubtitleEdit/subtitleedit/releases/download/${version}/SE${lib.replaceStrings [ "." ] [ "" ] version}.zip";
```

**Comment by paveloom** [comment]
Result of `nixpkgs-review pr 215877` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>subtitleedit</li>
  </ul>
</details>

File: pkgs/applications/video/subtitleedit/default.nix [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    description = "A subtitle editor";
+    homepage = "https://nikse.dk/subtitleedit/";
+    license = lib.licenses.gpl3Plus;
+    longDescription = ''
+      With Subtitle Edit you can easily adjust a subtitle if it is out of sync with
+      the video in several different ways. You can also use it for making
+      new subtitles from scratch (using the time-line /waveform/spectrogram)
+      or for translating subtitles.
+    '';
+    maintainers = [ lib.maintainers.paveloom ];
```
Comment (sikmir) [inline pkgs/applications/video/subtitleedit/default.nix:78]:
```suggestion
    maintainers = with lib.maintainers; [ paveloom ];
```

File: pkgs/applications/video/subtitleedit/default.nix [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+, mono
+, mpv
+, tesseract4
+}:
+
+let
+  makeBinPath = lib.makeBinPath;
+  makeLibraryPath = lib.makeLibraryPath;
+  replaceStrings = lib.replaceStrings;
+in
```
Comment (sikmir) [inline pkgs/applications/video/subtitleedit/default.nix:18]:
```suggestion
```

File: pkgs/applications/video/subtitleedit/default.nix [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+  src = fetchzip {
+    url = "https://github.com/SubtitleEdit/subtitleedit/releases/download/${version}/SE${replaceStrings [ "." ] [ "" ] version}.zip";
+    sha256 = "00w9jx704in3hbnzp0i7bhqkhbl0h5mahc5izwa980b67w08dc26";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  runtimeLibs = makeLibraryPath [
```
Comment (sikmir) [inline pkgs/applications/video/subtitleedit/default.nix:31]:
```suggestion
  runtimeLibs = lib.makeLibraryPath [
```

File: pkgs/applications/video/subtitleedit/default.nix [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    gtk2
+    hunspell
+    mpv
+    tesseract4
+  ];
+
+  runtimeBins = makeBinPath [
```
Comment (sikmir) [inline pkgs/applications/video/subtitleedit/default.nix:38]:
```suggestion
  runtimeBins = lib.makeBinPath [
```

File: pkgs/applications/video/subtitleedit/default.nix [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  makeBinPath = lib.makeBinPath;
+  makeLibraryPath = lib.makeLibraryPath;
+  replaceStrings = lib.replaceStrings;
+in
+stdenv.mkDerivation rec {
+  pname = "subtitleedit";
+  version = "3.6.11";
+
+  src = fetchzip {
+    url = "https://github.com/SubtitleEdit/subtitleedit/releases/download/${version}/SE${replaceStrings [ "." ] [ "" ] version}.zip";
```
Comment (sikmir) [inline pkgs/applications/video/subtitleedit/default.nix:24]:
```suggestion
    url = "https://github.com/SubtitleEdit/subtitleedit/releases/download/${version}/SE${lib.replaceStrings [ "." ] [ "" ] version}.zip";
```

File: pkgs/applications/video/subtitleedit/default.nix [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    homepage = "https://nikse.dk/subtitleedit/";
+    license = lib.licenses.gpl3Plus;
+    longDescription = ''
+      With Subtitle Edit you can easily adjust a subtitle if it is out of sync with
+      the video in several different ways. You can also use it for making
+      new subtitles from scratch (using the time-line /waveform/spectrogram)
+      or for translating subtitles.
+    '';
+    maintainers = with lib.maintainers; [ paveloom ];
+    platforms = lib.platforms.all;
```
Comment (sikmir) [inline pkgs/applications/video/subtitleedit/default.nix:65]:
```suggestion
    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
    platforms = lib.platforms.all;
```

**Pull request merged**